### ISSUE TITLE
add comments / clarifications / renames wrt direct tab messaging security

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#5328]
mostly comments and security clarifications, so we don't leave footguns around

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/pull/5320#pullrequestreview-1546188319_

**Comment by ioanmo226** [comment]
@sosnovsky I think this issue might be outdated now.
For example, in `sendToParentWindow`, we added `parentReference`, so it should no longer be unsafe.
Could you take a look and confirm?

```ts
private static sendToParentWindow(parentReference: ChildFrame, name: string, bm: Dict<unknown> & { messageSender?: Bm.Dest }, responseName?: string) {
```

**Comment by sosnovsky** [comment]
Yeah, should be safe now, as we're sending parent frame there, thanks for noticing!
